### DISCUSSION TITLE
Save SOC Dashboard custom OQL.

### DISCUSSION THREAD

**Discussion by medtemo** [Security-Onion-Solutions/securityonion#9537] [Ideas] [answered]
Hello!

I would like to reach out and ask if it may be possible to have a feature for saving  custom OQL queries in the SOC Dashboard for easy of reuse.

Thanks!

**Comment by brazorfajeje** [comment, marked as answer]
I think the features you are looking for is the one described in the Custom Queries section:

https://docs.securityonion.net/en/2.3/soc-customization.html?highlight=dashboard#custom-queries

**Reply by medtemo** [reply]
Thank you for quick reply.
